markdownhighlighter.h

```diff
@@ -27,6 +31,25 @@ QT_END_NAMESPACE
 class MarkdownHighlighter : public QSyntaxHighlighter {
     Q_OBJECT
 
+#ifdef QT_QUICK_LIB
+    Q_PROPERTY(QQuickTextDocument *textDocument READ textDocument WRITE
+                   setTextDocument NOTIFY textDocumentChanged)
+
+    QQuickTextDocument *m_quickDocument = nullptr;
```

Comment:
Not necessary to store this I think.

Reply:
It is, because of `inline QQuickTextDocument *textDocument() const { return m_quickDocument; };`

Reply:
right, thanks